packages/rule-components/src/ReportDetails/ReportDetails.css

```diff
@@ -0,0 +1,12 @@
+.OCMReportDetails {
+    padding: 20px 40px;
+}
+
+.OCMReportDetailsExternalLinkIcon {
+    margin-bottom: 1px;
+    margin-left: 6px;
+}
+
+.riskTitle {
```

Comment:
`ins-c-rule__risk-detail`

And it should be child style of `ins-c-rule__report-detail`